main.go

```diff
@@ -52,11 +52,12 @@ func main() {
 	flag.IntVar(&metricsAddr, "metrics-addr", 8443, "The address the default endpoints binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
-	flag.BoolVar(&options.CtrlConfig.EnableGangScheduling, "enable-gang-scheduling", false, "enable gang scheduling for workloads")
 	flag.StringVar(&options.CtrlConfig.GangSchedulerName, "gang-scheduler-name", "kube-batch", "specify the name of gang scheduler")
```

Comment:
we need to change the default value to be empty ?

Reply:
done